playground/app/views/books/index.html.erb

```diff
@@ -1,35 +1,57 @@
-<p id="notice"><%= notice %></p>
+<% content_for :title do %>Backend-search<% end %>
 
-<h1>Books</h1>
+<div class="container-fluid">
+  <h1>Backend-search</h1>
 
-<table>
-  <thead>
-    <tr>
-      <th>Title</th>
-      <th>Author</th>
-      <th>Publisher</th>
-      <th>Genre</th>
-      <th>Publication year</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
+  <div class="well">
+    A results page based on <code>Meilisearch Ruby Client</code> + <code>kaminari</code>.
+    <p>To see pagy pagination go to <%= link_to 'pagy', songs_path %></p>
 
-  <tbody>
-    <% @books.each do |book| %>
-      <tr>
-        <td><%= book.title %></td>
-        <td><%= book.author %></td>
-        <td><%= book.publisher %></td>
-        <td><%= book.genre %></td>
-        <td><%= book.publication_year %></td>
-        <td><%= link_to 'Show', book %></td>
-        <td><%= link_to 'Edit', edit_book_path(book) %></td>
-        <td><%= link_to 'Destroy', book, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+    <%= link_to 'New Book', new_book_path %>
+  </div>
 
-<br>
+  <div class="container my-5">
+    <div class="row">
+      <div class="col-sm-6 col-sm-offset-3 mx-auto">
+        <%= form_for Book.new, url: books_path, method: 'get' do |f| %>
+          <%= text_field :book, :q, class: "form-control", value: @q %>
+        <% end %>
+        <p>Press enter to search</p>
```

Comment:
Is there a specific reason why we don't have a search as you type? :)

Reply:
Since this is just a playground, I'm not sure if it is worthy of having such a configuration here, but we definitely can!

Also, in this case, we should have an instant search configurated environment. And connect directly over the meilisearch host instead of doing it by rails. It is a totally different way of implementation, usually rails developers does everything by the ruby side :|